packages/core/test/feedback/FeedbackWidgetManager.test.tsx

```diff
@@ -139,3 +139,69 @@ describe('FeedbackWidgetManager', () => {
     expect(getClient().getIntegrationByName(AUTO_INJECT_FEEDBACK_INTEGRATION_NAME)).toBeDefined();
   });
 });
+
+describe('FeedbackButtonManager', () => {
+
+  beforeEach(() => {
+    const client = new TestClient(getDefaultTestClientOptions());
+    setCurrentClient(client);
+    client.init();
+    consoleWarnSpy.mockReset();
+    resetFeedbackButtonManager();
+  });
+
+  it('showFeedbackButton displays the button when FeedbackWidgetProvider is used', () => {
+    const { getByText } = render(
+      <FeedbackWidgetProvider>
+        <Text>App Components</Text>
+      </FeedbackWidgetProvider>
+    );
+
+    showFeedbackButton();
+
+    expect(getByText('Report a Bug')).toBeTruthy();
+  });
+
+  it('hideFeedbackButton hides the button', () => {
+    const { queryByText } = render(
+      <FeedbackWidgetProvider>
+        <Text>App Components</Text>
+      </FeedbackWidgetProvider>
+    );
+
+    showFeedbackButton();
+    hideFeedbackButton();
+
+    expect(queryByText('Report a Bug')).toBeNull
```

Comment:
Missing call of the function.

````suggestion
    expect(queryByText('Report a Bug')).toBeNull()
````